pkg/exchange/bitget/stream.go

```diff
@@ -212,7 +225,19 @@ func parseEvent(in []byte) (interface{}, error) {
 		book.instId = event.Arg.InstId
 		return &book, nil
 
-	case ChannelTrade:
+	case strings.HasPrefix(string(ch), "candle"):
+		var kline KLineEvent
+		err = json.Unmarshal(event.Data, &kline.Events)
+		if err != nil {
+			return nil, fmt.Errorf("failed to unmarshal data into KLineEvent, Arg: %+v Data: %s, err: %w", event.Arg, string(event.Data), err)
+		}
+
+		kline.actionType = event.Action
```

Comment:
i think you can make these field public?

Reply:
The action type is used in the current package, so i made it private.